packages/ra-ui-materialui/src/detail/Create.tsx

```diff
@@ -226,6 +226,6 @@ const sanitizeRestProps = ({
     permissions = null,
     transform = null,
     ...rest
-}) => rest;
+}: any) => rest;
```

Comment:
I'm not a fan of this: setting a null default value should catch wrong type alerts, but we need typing here to catch true alerts for props that should not appear here.

Reply:
we already use any in most `sanitizeRestProps` functions

Reply:
Fixed it in all components